linkerd/app/outbound/src/http/logical/policy/route/metrics/labels.rs

```diff
@@ -5,17 +5,23 @@ use prometheus_client::encoding::*;
 use crate::{BackendRef, ParentRef, RouteRef};
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
-pub struct Route(pub ParentRef, pub RouteRef);
+pub struct HttpRoute {
+    parent: ParentRef,
+    route: RouteRef,
+    host: Option<String>,
+}
+
+#[derive(Clone, Debug, Hash, PartialEq, Eq)]
+pub struct GrpcRoute(pub ParentRef, pub RouteRef);
```

Comment:
:point_up: this is where we add a hostname to HTTP metrics, but intentionally refrain from doing so for our gRPC metrics.